audit/format.go

```diff
@@ -362,11 +363,12 @@ type AuditResponse struct {
 }
 
 type AuditAuth struct {
-	ClientToken string            `json:"client_token"`
-	Accessor    string            `json:"accessor"`
-	DisplayName string            `json:"display_name"`
-	Policies    []string          `json:"policies"`
-	Metadata    map[string]string `json:"metadata"`
+	ClientToken  string            `json:"client_token"`
+	Accessor     string            `json:"accessor"`
+	DisplayName  string            `json:"display_name"`
+	Policies     []string          `json:"policies"`
+	Metadata     map[string]string `json:"metadata"`
+	TokenNumUses int               `json:"token_num_uses,omitempty"`
```

Comment:
I'm not sure about the naming here. By definition, an Auth struct corresponds to a token, so it feels redundant -- otherwise why not have `token_policies` and `token_display_name`.